CONTRIBUTING.md

```diff
@@ -30,30 +30,96 @@ task “Capacity”
   entire team.
 
 
-## Kanban board
+## Sprint rituals
 
-We're currently using ZenHub for our [Kanban
-board](https://app.zenhub.com/workspaces/datagov-devsecops-579a2532d1d6ea9c3fcf5cfa/board).
+We follow two week sprints with the following rituals.
+
+- Daily standup (daily)
+- Sprint planning (first Monday of the sprint)
+- Sprint review and retrospective (last Friday of the sprint)
+
+
+## Story lifecycle
+
+**Stories** represent tactical increments of individually-valuable work
+deliverable by the team within a single iteration... often an isolated change in
+functionality aimed at achieving a goal for a particular kind of stakeholder,
+whether customer, user, or operator/admin. Stories are tracked on the [Kanban
+Board](https://app.zenhub.com/workspaces/datagov-devsecops-579a2532d1d6ea9c3fcf5cfa/board)
+and progress through these columns.
+
+- New
+- Icebox
+- Product backlog
+- Sprint backlog
+- In progress
+- Blocked
+- Ready for deploy
+- Done
 
 For Project Management Office and security compliance related tasks, see our
 [Incident Response repo](https://github.com/GSA/datagov-incident-response).
 
 
+### Definition of "Done"
+
+An agile "Definition of Done" (DoD) captures the team's agreed-upon standards
+for how we get work done at a consistent level of quality. Having a DoD ensures
+that non-functional requirements (NFRs) don't have to be re-litigated for every
+piece of work taken on, cards can be focused on just the relevant details, and
+new team members aren't surprised by assumed expectations of their colleagues.
+
+At our sprint reviews, we demo work that has reached the "Done" column and is of
+interest to our users or teammates.
+
+
+#### Column exit criteria
+
+Our DoD is broken up into a set of statements that should be true
+for each card before it moves to the next column on the board.
+
+Before advancing a card from one column to the next on the board, it should meet
+the "exit criteria" for the current column, which are listed below.
+
+
 ### Columns
 
+
 #### New
 
 New issues that need to be triaged.
 
 
+##### Exit criteria
+
+- Relevant points from any discussion in the comments is captured in the initial
+  post.
+- Decision is made to move to the Backlog or Icebox columns, or close.
+
+
 #### Icebox
 
 Work that has been de-prioritized.
 
 
-#### Product backlog
+##### Exit criteria
 
-Work that we are planning on doing and will schedule into a sprint.
+- When reviewing priorities, we may pull items out of the Icebox.
+- Items move to Product backlog for grooming.
+
+
+#### Product Backlog
+
+Work that we are planning on doing and will groom and schedule into a sprint.
+
+
+##### Exit criteria
+
+- Indicate the intended benefit and who the story is for in the "as a ..., I want
+  ..., so that ..." form.
+- Acceptance criteria is defined.
+- If necessary, the story includes a security testing plan. For example, the
+  acceptance criteria include automated tests and alerts for unexpected behavior.
```

Comment:
Security testing seems orthogonal to automated tests and alerts. Maybe split this into two items?

Reply:
Hmm, what I'm trying to say is, "we've thought about the security implications of this feature and have included any tasks to address those implications in the acceptance criteria. For example, if we need to assert that a URL is accessible only by authorized users, then an acceptance criteria is added to add an automated test that will test that URL."

so... maybe.. this?

> If necessary, the story includes a security testing plan and any tasks from that plan are included as acceptance criteria.

Reply:
Sure, that's clearer.

Reply:
Updated, ready for another review.